templates/availability/_table.html.twig

```diff
@@ -50,7 +50,7 @@
             <tr>
                 <td class="hour-title">{{ choices[i].date|date('H') }}-{{ choices[i+1].date|date('H') }}h</td>
                 {% for j in 0..6 %}
-                    <td class="slot-box" data-day="{{ j }}" data-from="{{ choices[i+1].date|date('U') }}" data-to="{{ choices[i+1].date|date('U') }}">
+                    <td class="slot-box" data-day="{{ j }}" data-from="{{ choices[j*nbChoicesPerDay+i].date|date('U') }}" data-to="{{ choices[j*nbChoicesPerDay+i].date|date_modify('+2 hours')|date('U') }}">
```

Comment:
are you sure we cannot have this information without hardcoding `2 hours` here ?

Reply:
Will be resolved by https://github.com/crf-devs/resop/issues/129